src/lsp/cobol_preproc/src_overlay.ml

```diff
@@ -91,8 +91,9 @@ let limits: manager -> srcloc -> limit * limit = fun ctx loc ->
     | Some lexloc -> lexloc
     | _ -> Limit.make_virtual (), Limit.make_virtual ()
   in
-  Links.replace ctx.right_of s (loc, e);  (* Replace to deal with duplicates. *)
```

Comment:
Could u take this opportunity to rename `e` in `right` and `s` in `left`, or some other **long** names ?